ansible/library/check_bgp_ipv6_routes_converged.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python
+
+from ansible.module_utils.basic import AnsibleModule
+import json
+import time
+import logging
+import datetime
+from ansible.module_utils.debug_utils import config_module_logging
+import gzip
+import base64
+
+
+def get_bgp_ipv6_routes(module):
+    cmd = "docker exec bgp vtysh -c 'show ipv6 route bgp json'"
```

Comment:
Can we redirect this to a file and then load it rather than using stdout and check if this makes things faster?